ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -334,7 +337,11 @@ open class SideTabBar: UIView {
     }
 
     @objc private func handleAvatarViewTapped(_ recognizer: UITapGestureRecognizer) {
-        delegate?.sideTabBar?(self, didActivate: avatarView!)
+        guard let avatar = avatar else {
+            return
```

Comment:
Did you want to resolve the other two optionals here too?

Reply:
I only want to validate the avatar because I don't want to potentially call the delegate callback with a nil instance.
Ideally we shouldn't have this optional func on the protocol and should provide a default implementation as the pattern suggests: https://docs.swift.org/swift-book/LanguageGuide/Protocols.html

We can leave the optional chaining for both the delegate instance and the optional func in the protocol here.